### PR TITLE
mantaflow 0.9 (new formula)

### PR DESCRIPTION
**Pull request by sebbas**
mantaflow is an open-source framework targeted at fluid simulation research in computer graphics.

### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

File: mantaflow.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class Mantaflow < Formula
+  desc "Open computer graphics fluid simulation framework"
+  homepage "http://mantaflow.com/"
+  url "http://mantaflow.com/download/manta-src-0.9.tar"
+  sha256 "f55298f10650b35454f3134d2d0e265b19c24c3c68991e4bbedf5a460be67077"
```
Comment (sebbas) [inline mantaflow.rb]: Ah, yeah - I just added that

File: mantaflow.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    system "make"
+
+    bin.install "manta"
+
+    # Symlink 'mantaflow' to avoid confusion when getting started with mantaflow
+    # Actual command is 'manta'
+    bin.install_symlink "manta" => "mantaflow"
+
+    # Copy the python sample scene files
+    pkgshare.install Dir["scenes/*"] if build.with? "scenes"
```
Comment (sebbas) [inline mantaflow.rb]: Yes, probably better to remove the extra scene flag and just install scripts by default. I just committed that.

File: mantaflow.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  option "with-openmp", "Build with OpenMP support"
+  option "with-qt5", "Build the QT GUI version"
+  option "with-scenes", "Install python sample scene files"
+
+  depends_on "cmake" => :build
+  depends_on "qt5" => :optional
+  depends_on :python => :build
+
+  needs :openmp if build.with? "openmp"
```
Comment (sebbas) [inline mantaflow.rb]: I think it's better to make OpenMP optional for now. The official Manta CMake setup also leaves it disabled by default.

**Comment by sebbas** [comment]
Hi @jonchang! I just committed a bunch of improvements - see comments for that. 
With the help of Linuxbrew is was also able to fix an issue with C++11 - Travis CI is now passing as well. What's left is CircleCI, could you take a look at that?

**Comment by sebbas** [comment]
@jonchang Never mind, I think my last commit fixed the issue :) (python dependency wasn't setup correctly). What do you think?

**Comment by jonchang** [comment]
This is merged. Thanks @sebbas for contributing to Homebrew!

**Comment by sebbas** [comment]
@jonchang Awesome, thanks for merging!

**Pull request closed without merging**